lib/generator/template_values.rb

```diff
@@ -0,0 +1,48 @@
+module Generator
+  # Contains methods accessible to the ERB template
+  class TemplateValues
+    attr_reader :sha1, :version, :test_cases
+
+    def initialize(sha1:, version:, test_cases:)
+      @sha1 = sha1
+      @version = version
+      @test_cases = test_cases
+    end
+
+    def get_binding
+      binding
+    end
+  end
+
+  # This is not a standalone module, is that a problem?
+  module TemplateValuesFactory
```

Comment:
Are the `TODO`s in this module meant to be completed after merging?

I think it's completely reasonable if so, especially since these changes make the process of generating test sooooooo much more pleasant (`--freeze` 😍).

Just wanted to double check that these didn't get missed 😄